Changes

```diff
@@ -226,6 +226,14 @@ Working version
 
 ### Bug fixes:
 
+- #13691: Make four globals underlying Gc.control atomic to avoid C data
+  races against them
+  (Jan Midtgaard, review by Miod Vallat)
+
+- #13408: Fix misplaced debug runtime assertion triggerable by a race
+  between domain exit and backup thread
+  (Miod Vallat and Gabriel Scherer, report by Jan Midtgaard)
+
```

Comment:
Is there a reason this contains Changes from a different PR?

Reply:
Sorry, my bad - something must have gone wrong when rebasing :shrug:

Reply:
I've force-pushed an updated Changes entry in 3b43103